server/routes/api/ads.js

```diff
@@ -134,4 +134,28 @@ router.post('/', async (req, res) => {
     }
 });
 
+router.post('/close/:id', async (req, res) => {
```

Comment:
Actually this should be a `PUT` because your'e not creating a new one your'e just updating

Reply:
change it to `put` function